content/en/graphing/metrics/distributions.md

```diff
@@ -10,33 +10,38 @@ further_reading:
     text: "Using Distributions in DogStatsD"
 ---
 
-<div class="alert alert-warning">
-This feature is in beta. <a href="/help">Contact Datadog support</a> to enable distribution metrics for your account.
-</div>
-
 ## Overview
 
-Distributions are a [metric type][1] that can be thought of as a global version of the [Histogram metric][2], which measures the statistical distribution of discrete values on a single host. Distributions observe the values that are sent from multiple hosts to measure statistical distributions across your entire infrastructure, allowing you to compute global percentiles across your entire dataset.
+Distributions are a metric type in Agent 6 that aggregate the values that are sent from multiple hosts during a flush interval to measure statistical distributions across your entire infrastructure server-side.  This can be thought of as a global version of the [Histogram metric][1], which measures at the Agent-level the statistical distribution of values on a single host.
```

Comment:
```suggestion
Distributions are a metric type in Agent v6 that aggregate the values that are sent from multiple hosts during a flush interval to measure statistical distributions across your entire infrastructure server-side.  This can be thought of as a global version of the [Histogram metric][1], which measures at the Agent-level the statistical distribution of values on a single host.
```
Does the customer need a specific Agent version or higher (example v6.12+)?